### DISCUSSION TITLE
This app can't be run on this device

### DISCUSSION THREAD

**Discussion by golephish** [valinet/ExplorerPatcher#3089] [Q&A] [answered]
![Screenshot 2024-04-04 122229](https://github.com/valinet/ExplorerPatcher/assets/48928387/776bcb75-6ee6-46ab-a7c6-0597f30bdef4)

on canary, 26100.1

no matter what I've tried it will not launch. disabling program compatibility service was the only thing I could think of. any ideas?

**Comment by mackid1993** [comment, marked as answer]
Try renaming the installer. They did this to StartAllBack too. Perhaps they are going after Explorer Patcher in the same way. See here:
https://msfn.org/board/topic/183013-startallback-for-windows-11/?do=findComment&comment=1262502

**Comment by pyrates999** [comment]
Canary builds are not supported because the code is not final and can change with each new release.

**Comment by mackid1993** [comment]
Obviously not but 26100 is the dev build as well and likely 24H2 RTM. Microsoft is arbitrarily blocking both StartAllBack and Explorer Patcher. It's important that this is known so the devs of both apps can mitigate this before general availability.

**Reply by pyrates999** [reply]
In other words, don't be installing canary builds of Windows 11 unless you are a beta tester and will be reporting bugs you find with it.  Installing EP and StartAllBack modifies the test environment.  And Microsoft doesn't want you modifying the test environment you twit.

**Reply by mackid1993** [reply]
No need to call people twits. I don't even use Explorer Patcher and I'm running stable Windows. I was just being helpful.

**Comment by golephish** [comment]
I realize that canary builds are bleeding edge, I was just trying to pass along the information. I appreciate all the help and replies  : )

**Comment by mackid1993** [comment]
No problem. I appreciate not being insulted by randos when just trying to help and especially not even being a user of the software or on an Insider build.

**Comment by MidnightInAurora** [comment]
Not to sound rude, but I don't really give a rats ass about what Microsoft wants.  EP is broken and it doesn't matter if the MS product is Home/Pro/Ent or Server.  My guess is that Explorer.exe and the supporting DLLs are the same across each for every equivalent build.

If EP wants to be a first-choice product, I'd suggest they get off their collective butts and fix the various issues, not the least of which is the missing system tray.  Other shell enhancers don't seem to have a problem with 26100 and the system tray, even on Server 2025 26085, and EP shouldn't either.

Stop making excuses and get busy.

My 2 cents.

**Reply by Anamon** [reply]
Comments like this one are what makes developers want to stop working on free software altogether.

They don't owe anybody anything. Their goal isn't to make anybody's "first-choice product". If it's helpful, use it and be grateful. If you think other tools are better – use those.

**Comment by golephish** [comment]
renaming it to anything else will allow it to run.  as I say was just trying to pass along my findings. thank you to all for the responses.
